plugins/packages/appwrite/lib/index.ts

```diff
@@ -0,0 +1,91 @@
+import { QueryError, QueryResult, QueryService, ConnectionTestResult } from '@tooljet-plugins/common';
+import { SourceOptions, QueryOptions } from './types';
+import sdk from 'node-appwrite';
+import { bulkUpdate, createDocument, deleteDocument, getDocument, queryCollection, updateDocument } from './operations';
+
+export default class Appwrite implements QueryService {
+  async run(sourceOptions: SourceOptions, queryOptions: QueryOptions, dataSourceId: string): Promise<QueryResult> {
+    const database = await this.getConnection(sourceOptions);
+    const operation = queryOptions.operation;
+    let result = {};
+
+    try {
+      switch (operation) {
+        case 'list_docs':
+          result = await queryCollection(
+            database,
+            queryOptions.collectionId,
+            queryOptions.limit,
+            queryOptions.order_fields,
+            queryOptions.order_types,
+            queryOptions.where_field,
+            queryOptions.where_operation,
+            queryOptions.where_value
+          );
+          break;
+        case 'get_document':
+          result = await getDocument(database, queryOptions.collectionId, queryOptions.documentId);
+          break;
+        case 'add_document':
+          result = await createDocument(
+            database,
+            queryOptions.collectionId,
+            typeof queryOptions.body === 'string' ? JSON.parse(queryOptions.body) : queryOptions.body
+          );
```

Comment:
Try to use json5 like
https://github.com/ToolJet/ToolJet/pull/2590/files#diff-53b0ae5e2470db40e970c6a0359f091a01a92175f9eb3f0af8745cd5e8895af4R158